src/Api/Service/Sf.php

```diff
@@ -763,6 +778,23 @@ public function lex_comment_updateStatus($commentId, $status)
         return LexCommentCommands::updateCommentStatus($this->projectId, $commentId, $status);
     }
 
+    public function lex_comment_getByWord($wordId)
+    {
+        $project = new LexProjectModel($this->projectId);
```

Comment:
If you look at most of the other methods in this API they are a one line call to another method (usually a command). This one belongs in **LexCommentCommands**. The reason for that is this file is particularly difficult to test well where as the thing it calls (the command) is easy to test. When you move it to **LexCommentCommands** make sure you have added tests for it in **LexCommentCommandsTest.php**.

This applies to all methods you have introduced below.